src/rebar_file_utils.erl

```diff
@@ -37,11 +39,25 @@
          reset_dir/1]).
 
 -include("rebar.hrl").
+-include_lib("providers/include/providers.hrl").
 
 %% ===================================================================
 %% Public API
 %% ===================================================================
 
+try_consult(File) ->
+    case file:consult(File) of
+        {ok, Terms} ->
+            Terms;
+        {error, enoent} ->
+            [];
+        {error, Reason} ->
+            throw(?PRV_ERROR({bad_term_file, File, Reason}))
```

Comment:
Ah yeah I see you made it so the module prints the error itself. Nevermind then. I skipped that bit.